session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -913,6 +914,27 @@ public SessionDataSet executeQueryStatement(String sql)
         execResp.isIgnoreTimeStamp());
   }
 
+  public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, long endTime)
+          throws StatementExecutionException, IoTDBConnectionException {
+
+    String statement = "raw data query";
+    TSRawDataQueryReq execReq = new TSRawDataQueryReq(sessionId, paths, startTime, endTime, statementId, "");
```

Comment:
as the statement parameter is empty, why we need the parameter?

Reply:
Hi Houliang, thank you for pointing out. I will pass the statement string to parameter. We will need statement and statementId in closeSession() when executing multiple queries.